tests/Builder/DatagridBuilderTest.php

```diff
@@ -62,8 +62,12 @@ protected function setUp(): void
         );
 
         $this->admin = $this->createMock(AdminInterface::class);
-        $this->modelManager = $this->createMock(ModelManager::class);
-
+        $this->modelManager = $this->getMockBuilder(ModelManagerInterface::class)
+            ->setMethodsExcept([])
+            // NEXT_MAJOR: remove the "hasMetadata" and "getParentMetadataForProperty" methods and just mock the interface normally
```

Comment:
Why the need to add a NEXT_MAJOR if you're already on the master branch ?

Reply:
its only needed for some tests that are not cleaned up yet (still with `NEXT_MAJOR` comments). I did not want to do it all in one big PR

Reply:
See https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/90c4d7d59d58e5a7199ce6f6f9015cdaefab7623/tests/Builder/DatagridBuilderTest.php#L136

Reply:
@VincentLanglet removed since you did the cleanup on master already :+1: